supertokens_python/recipe/emailverification/utils.py

```diff
@@ -35,8 +35,8 @@ async def func(_: User, __: Dict[str, Any]):
 
 def default_create_and_send_custom_email(app_info: AppInfo) -> Callable[[User, str, Dict[str, Any]], Awaitable[None]]:
     async def func(user: User, email_verification_url: str, _: Dict[str, Any]):
-        if ('SUPERTOKENS_ENV' not in environ) or (
-                environ['SUPERTOKENS_ENV'] != 'testing'):
+        if ('SUPERTOKENS_ENV' in environ) and (
+                environ['SUPERTOKENS_ENV'] == 'testing'):
```

Comment:
@bhumilsarvaiya Just fyi, I have fixed this error in my PR as well.

Reply:
👍🏼